docs/eks-installation.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Antrea in AWS EKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an AWS EKS cluster.
+
+Assuming you already have an EKS cluster, and have KUBECONFIG environment variable pointing to<br>
+the kubeconfig file of that cluster.
+
+Based on EKS worker Node MTU size and service CIDR, adjust defaultMTU and serviceCIDR values of<br>
+antrea-agent.conf in ./build/yamls/antrea-eks.yml accordingly, and apply<br>
+./build/yamls/antrea-eks.yml to the EKS cluster.
+
+```bash
+kubectl apply -f ./build/yamls/antrea-eks.yaml 
+```
+
+Now Antrea should be plugged into the EKS CNI and is ready to enforces NetworkPolicy.<br> 
+You can validate it either manually or by running NetworkPolicy tests as follows:
+
+```bash
+./ci/run-k8s-e2e-tests.sh --e2e-network-policy
```

Comment:
This is not for end users. Suggest to remove this part.

Maybe you can refer to getting-started.md on how to check Antrea components are running (there is a gif there).

Reply:
done.